test/jdk/javax/net/ssl/SSLEngine/Basics.java

```diff
@@ -26,8 +26,7 @@
  * @bug 4495742
  * @summary Add non-blocking SSL/TLS functionality, usable with any
  *      I/O abstraction
- * @ignore JSSE supported cipher suites are changed with CR 6916074,
- *     need to update this test case in JDK 7 soon
+ * @ignore 8298867
```

Comment:
Instead of using ignore tag, I would prefer to put these tests in ProblemList with these bug numbers.

Reply:
Moved bugs to ProblemList and removed ignore tags